### ISSUE TITLE
SSL on the site

### ISSUE THREAD

**Issue by Badlapje** [sakamies/vocabs#25]
Love the resource, but would be great if you enabled SSL on the site.

**Comment by sakamies** [comment]
Indeed! I don't have much time to work on this at the moment but I'll try to change servers so the new one will have ssl. Thanks!
